iktomi/db/sqla/types.py

```diff
@@ -56,6 +56,8 @@ class Html(HtmlBase):
     '''
 
     def __init__(self, _impl, markup_class=None):
+        # callable is need to be able assign parametrized classes, for example
```

Comment:
Something like “SQLAlchemy traditionally allows both class and instance as type (`String` and `String(100)`)” should be enough.